File: lang-kotlin/src/test/resources/fixtures/repeated-collection-iteration/negative/different-sources.kt
class ReportBuilder {
    fun buildReport(orders: List<Order>, customers: List<Customer>) {
        val totals = mutableListOf<Double>()
        for (order in orders) {
            totals.add(order.total)
        }
        val names = mutableListOf<String>()
        for (customer in customers) {
            names.add(customer.name)
        }
    }
}

File: lang-javascript/src/test/resources/fixtures/repeated-collection-iteration/negative/different-sources.js
function mergeLists(users, orders) {
    const names = [];
    for (const user of users) {
        names.push(user.name);
    }

    const totals = [];
    for (const order of orders) {
        totals.push(order.amount);
    }

    return { names, totals };
}
